src/messages/Whip_InvalidVersionRequirementMessage.php

```diff
@@ -16,15 +16,15 @@ class Whip_InvalidVersionRequirementMessage implements Whip_Message {
 	private $requirement;
 
 	/**
-	 * @var
+	 * @var int
 	 */
 	private $detected;
 
 	/**
 	 * Whip_InvalidVersionRequirementMessage constructor.
 	 *
 	 * @param Whip_VersionRequirement $requirement
-	 * @param                         $detected
+	 * @param int                     $detected
```

Comment:
As this parameter is holding a `version` number (4.1.2), the type of the variable should be `string`